### PR TITLE
Add some missing shortcuts to the readme

### PR DESCRIPTION
**Pull request by ssgelm**
I noticed the readme was missing a few things I use regularly so I figured it was worth updating it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,10 +59,11 @@ g CTRL+] - pop up a selector if there is more than one definition; if there is o
 \rt - rebuild tags
 ```
 
-Git Grep - Fugitive
+Git - Fugitive
 
 ```
 \gw - git grep for the word under the cursor
+\gh - get the github url for the current file
```
Comment (pgr0ss) [inline README.md]: Neither of these use Fugitive, so maybe the heading should just be `Git`?

File: README.md [hunk 2]
```diff
@@ -78,6 +80,7 @@ Ruby Specific Mappings
 \rf - run the spec under the cursor
 \rl - run the last spec
 \rs - syntax check the current file
+Ctrl+L (in insert mode) - insert a " => " at the current cursor position
```
Comment (pgr0ss) [inline README.md]: Lowercase `l`, right?

File: README.md [hunk 3]
```diff
@@ -87,3 +90,8 @@ Python-Specific Mappings
 \rf - run the current test
 \rl - run the last test
 ```
+
+Searching
+
+```
```
Comment (pgr0ss) [inline README.md]: I think you forgot to close these quotes.

File: README.md (hunk 2 above)
Comment (ssgelm) [reply README.md]: Yeah, though that felt more confusing to me... I think people largely understand it's lowercase cause there's no `Shift` there?

File: README.md [hunk 4]
```diff
@@ -58,10 +59,11 @@ g CTRL+] - pop up a selector if there is more than one definition; if there is o
 \rt - rebuild tags
 ```
 
-Git Grep - Fugitive
+Git - Fugitive
```
Comment (ssgelm) [inline README.md]:
```suggestion
Git
```

File: README.md (hunk 1 above)
Comment (ssgelm) [reply README.md]: Oops, I meant to change that and neglected to.  Fixed

File: README.md (hunk 3 above)
Comment (ssgelm) [reply README.md]: That I did!  Fixed.

File: README.md (hunk 2 above)
Comment (pgr0ss) [reply README.md]: I assume capital letters in shortcuts imply a shift, but I don't know if that's universal.

**Pull request merged**